front/pages/poke/[wId]/memberships.tsx

```diff
@@ -82,7 +82,7 @@ const MembershipsPage = ({
       if (!r.ok) {
         throw new Error("Failed to revoke user.");
       }
-      await router.reload();
+      router.reload();
```

Comment:
So all along this wasn't returning a promise. Should probably have a linter to prevent awaiting things that aren't promises

Reply:
Yep